internal/widget/separator.go

```diff
@@ -0,0 +1,99 @@
+package widget
+
+import (
+	"image/color"
+
+	"fyne.io/fyne"
+	"fyne.io/fyne/canvas"
+	"fyne.io/fyne/theme"
+)
+
+var _ fyne.Widget = (*Separator)(nil)
+
+// Separator is a widget for displaying a separator with themeable color.
+type Separator struct {
+	Base
+	colorFn func() color.Color
+	minSize fyne.Size
+}
+
+// NewSeparator creates a separator with the given min size and the color delivered by the given callback.
+func NewSeparator(minSize fyne.Size, colorFn func() color.Color) *Separator {
+	return &Separator{minSize: minSize, colorFn: colorFn}
+}
+
+// CreateRenderer returns a new renderer for the separator.
+//
+// Implements: fyne.Widget
+func (s *Separator) CreateRenderer() fyne.WidgetRenderer {
+	bar := canvas.NewRectangle(theme.DisabledTextColor())
```

Comment:
I think the reason the images are failing is that widgets are using ShadowColor() for this at the moment

Reply:
Sorry, I don’t know which tests you refer to.

Reply:
Quite right they are not failing sorry. The reason you had to replace a lot of images is that the divider is using shadow colour in most places. Isn’t that more logical than the text disabled colour in menu?